InOut/CMakeLists.txt

```diff
@@ -9,55 +9,48 @@ option(USE_PORTMIDI "Build the PortMIDI I/O module" ON)
 option(USE_IPMIDI "Build the IPMIDI I/O module" ON)
 option(USE_JACK "Build the jack I/O module and opcodes" ON)
 option(USE_ALSA "Build the ALSA I/O module" ON)
-# option(USE_COREAUDIO "Build the CoreAudio I/O module" ON)
-# option(USE_COREMIDI "Build the CoreMIDI I/O Module" ON)
-# option(USE_AUDIOUNIT "Build the CoreAudio AudioUnit I/O module (requires CoreAudio)" ON)
+option(USE_COREMIDI "Build the CoreMIDI I/O Module" ON)
+option(USE_AUDIOUNIT "Build the CoreAudio AudioUnit I/O module (requires CoreAudio)" ON)
```

Comment:
Why enable this again?

Reply:
Ok, I set it back to off by default. Seems a shame though if the plugins still work

Reply:
Wait, nevermind, they were enabled before because we weren't checking for `USE_COREMIDI` and `USE_AUDIOUNIT`. But we need to have the variables so that `check_deps` will work. So reenabled.

Reply:
Ok so. I wasn't insinuating that they should be disabled. I don't know why they were disabled before.